src/main.js

```diff
@@ -1,14 +1,37 @@
 'use strict';
 
 const API_KEY = 'e9979803c3894fa182b399c482279297';
-const HOST_URL = 'https://newsapi.org/v2';
-const SOURCES_PARAM = '/top-headlines';
-
+const HOST_URL = 'https://newsapi.org/v2/';
+const SOURCES_PARAM = 'top-headlines';
 const url = `${HOST_URL}${SOURCES_PARAM}?country=us&apiKey=${API_KEY}`;
 const req = new Request(url);
+const container = document.getElementById('container');
+const fragment = document.createDocumentFragment();
+const getRandomIntNumber = (min, max) => Math.floor(Math.random() * (max - min + 1)) + min;
+const getRandomImageUrl = () => `./src/img/news-${getRandomIntNumber(1, 3)}.jpg`;
+const loader = document.getElementById("loader");
+
+window.onload = () => loader.style.display = "none";
+
 fetch(req)
-  //.then(response => response.json()).then(myJson => console.log(JSON.stringify(myJson)))
-   .then(response => {
-   console.log(response.json());
-   })
-  .catch(error => console.log(error) );
+  .then(response => response.json())
+  .then(response => {
+    response.articles.forEach((article) => {
```

Comment:
What if `articles` prop is not populated in response?

Reply:
Fixed.